mtp_noms_ops/templates/security/accept_or_reject_check.html

```diff
@@ -128,21 +132,10 @@ <h3 class="heading-medium">{% trans 'Credit decisions matching debit card or pri
           </tr>
           <tr class="mtp-check-description-row">
             <td></td>
-            <td colspan="4"><b>Description:</b> {{ credit.security_check.description }}</td>
-          </tr>
-          <tr class="mtp-check-rules-row">
-            <td></td>
-            <td colspan="4"><b>Rules:</b>
-              <ul>
-                {% for rule in credit.security_check.rules %}
-                  <li> {{ rule }} </li>
-                {% endfor %}
-              </ul>
-            </td>
-          </tr>
-          <tr class="mtp-check-decision-row">
-            <td></td>
-            <td colspan="4"><b>Decision Details:</b>
+            <td colspan="4">
+              <strong>Rule:</strong> {{ credit.security_check.description }}
+              <br />
+              <strong>Decision Details:</strong>
```

Comment:
the text should be marked for translation and this element should be inside the if-block below

Reply:
Translation yes, but if you look at the else clause of the if block i've added an alternate text to add clarity

Reply:
you're right i _did_ miss the else block. i was comparing it to the merged history list page which has no else clause. they ought to look the same however, so we should add the else to that too.